docs/astria.md

```diff
@@ -0,0 +1,140 @@
+# [Astria](https://www.astria.org/): Decentralized by Default
+ - [The Astria dev-cluster](#the-astria-dev-cluster)
+ - [Architecture](#architecture)
+    - [Users](#users)
+    - [Rollups](#rollups)
+    - [Composer (under construction)](#composer-under-construction)
+    - [The Shared Sequencer](#the-shared-sequencer)
+    - [Relayer](#relayer)
+    - [Data Availability](#data-availability)
+    - [Conductor](#conductor)
+ - [Running the Cluster](#running-the-cluster)
+    - [Installation](#installation)
+    - [Deployment](#deployment)
+    - [Faucet and Block Explorer](#faucet-block-explorer-and-test-data)
+    - [Multiple Rollups](#run-multiple-rollups)
+    - [What's Going on Under the Hood?](#whats-going-on-under-the-hood)
+ - [Summary](#summary)
+
+## The Astria [dev-cluster](https://github.com/astriaorg/dev-cluster)
+Astria's Shared Sequencer Network allows multiple rollups to share a single decentralized network of sequencers that’s permissionless to join. This shared sequencer network provides out-of-the-box censorship resistance, fast block confirmations, and atomic cross-rollup inclusion guarantees.
+
+The Astria dev-cluster is the collective stack of all of Astria's components packaged together using Kubernetes. While we generally refer to Astria as the network of shared sequencers, we provide several other components to make it simpler to integrate with the shared sequencer network. The dev-cluster is provided to make developing and testing the Astria network, as well as integrations with Astria, as simple as possible.
+
+This document outlines what each of the components in the dev-cluster do, and provides instructions for how to run the dev cluster locally.
+
+## Architecture
+At a high level, Astria breaks out all of the components that are historically combined together in monolithic blockchains into individual components. This results in a modular ecosystem that allows users to swap out components to fit their specific needs.
+
+![Astria Architecture](images/astria-architecture.png)
+Astria's architecture follows the structure outlined in the first diagram. Notable differences between Astria and the basic structure are the Composer, Relayer, and Conductor. Astria's Composer handles the order flow operations, and the Relayer and Conductor facilitate the soft and firm commitments. Ultimately, Astria's architecture is just an expanded version of the first diagram.
```

Comment:
>  Astria's architecture follows the structure outlined in the first diagram. Notable differences between Astria and the basic structure are the Composer, Relayer, and Conductor.

Kinda confusing? Maybe we should call this an Decentralized Rollup w/ Astria Architecture? I don't think notable differences part makes much sense, notable differences from what?

Reply:
This has been reworded.